Periodic_2_triangulation_2/doc/Periodic_2_triangulation_2/Periodic_2_triangulation_2.txt

```diff
@@ -199,7 +199,6 @@ parameters:
   `Periodic_2TriangulationTraits_2` (for `Periodic_2_triangulation_2`)
   and `Periodic_2DelaunayTriangulationTraits_2` (for
   `Periodic_2_Delaunay_triangulation_2`) in the reference manual and
```

Comment:
```suggestion
  `Periodic_2_Delaunay_triangulation_2`).
```

Reply:
I agree, though it is not up to me to decide to change the texts (unless it is obviously wrong).
Against the change is that when printing (and it is a general text so this might happen) there is no direct link to the reference manual (although I expect that the later is valid on many more places).

Reply:
I don't think it is very likely that someone will ever print the P2T2 user manual, and concepts and classes are used without reminding every time the reader that they are part of the reference manual.

Also I am a maintainer of the periodic triangulations, so it is ok to apply the suggestion and thanks for the patch.